app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,134 @@
+running in the *default* group, but with application registration disabled.
+
+* Another previous version of the Service, published to the Dev Portal, and
+running in a *custom* runtime group. This version would only have the documentation
+available, and developers would not have access to application registration for
+this version.
+
+Out of these three scenarios, only the Service version in the first scenario
+would be available for application registration.
+
+## Multiple runtime groups
+{:.badge .enterprise}
+
+In addition to the default runtime group, you can also configure two additional
+custom groups. Splitting runtime instances and configuration into multiple
+runtime groups provides multiple specialized environments in one account.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+[Set up and manage runtime groups &rarr;](/konnect/configure/runtime-manager/runtime-groups/manage)
+
+## Administration
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions.
+
+[Administer teams &rarr;](/link)
+
+## Global configuration
+
+For each runtime group, you can spin up runtime instances and configure global
+options for the following {{site.base_gateway}} objects:
+* Consumers
+* Plugins
+* Upstreams
+* Certificates
+* SNIs
+
+A **global** object is a set of configurations that apply to, or can be used
+by, all objects in a runtime group. For example, if you set up a Proxy Caching
+plugin in the default runtime group and set it to `Global`,
+the plugin configuration will apply to all Service versions in the group.
+
+When there are multiple runtime groups, configuration for any of the global
+objects only applies to the runtime group that it was created in. Consumers and
+their authentication mechanisms don't carry over to other runtime groups.
+
+[Manage global entities &rarr;](/konnect/configure/runtime-manager/manage-entities)
+
+## References
+
+### Runtime groups table
+
+The following describes the content of the table found on the [Runtime Manager overview](https://konnect.konghq.com/runtime-manager/).
+
+Column | Description
+-------|-------------
+Name | The name of the runtime group. This group name must be unique in the organization.
+ID | A unique ID for the group. This ID is automatically generated when a group is created.
+Type | Types of runtimes that this group contains. Currently, the only supported runtime type is `Kong Gateway`.
+Active instances | The number of runtime instances in this group that are `Connected`, `Normal`, or `In Sync`.
```

Comment:
Do we have these statuses documented anywhere?

Reply:
Not only do we not have them documented, I'm not sure they'll even exist. @haydenlam currently there's nothing like this in the preview environment; will there be an Active Instances column?

Reply:
If it's not there by tomorrow (the 30th), let's remove this line